### PR TITLE
Coming soon simplified (WIP prototype)

### PR DESCRIPTION
**Pull request by simison**
#### Changes proposed in this Pull Request

* Add simple coming soon to FSE plugin:

#### Testing instructions

1. Apply D45700-code in your sandbox and make sure you're sandboxing the API.
1. Build Calypso or use live-link
1. Create new page with "coming soon" texts
1. Go to "pages" in Calypso, and click three dots menu for that page
1. Set the page as "coming soon"
1. Build FSE plugin and sync it to your sandbox: `cd apps/full-site-editing && yarn dev --sync`
1. Go to your site's frontpage as non-authenticated user, confirm you now see your coming soon page
1. Go to same page as authenticated user, you should see the page normally
1. In editor you can edit everything regularly
1. Customizer loads homepage regurlarly

### CODE REVIEW

File: client/my-sites/pages/page-card-info/index.jsx [hunk 1]
```diff
@@ -66,6 +67,12 @@ function PageCardInfo( {
 						includeBasicStatus={ true }
 					/>
 				) }
+				{ isComingSoon && (
+					<span className="page-card-info__badge">
+						<Gridicon icon="house" size={ ICON_SIZE } className="page-card-info__badge-icon" />
+						<span className="page-card-info__badge-text">{ translate( 'Coming soon page' ) }</span>
```
Comment (a8ci18n) [inline client/my-sites/pages/page-card-info/index.jsx:73]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'Coming Soon!' )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/coming-soon-simplified)

File: client/my-sites/pages/page/index.js [hunk 2]
```diff
@@ -346,10 +395,13 @@ class Page extends Component {
 		}
 
 		return (
-			<PopoverMenuItem onClick={ this.exportPage }>
-				<Gridicon icon="cloud-download" size={ 18 } />
-				{ this.props.translate( 'Export page' ) }
-			</PopoverMenuItem>
+			<>
+				<MenuSeparator key="separator" />
```
Comment (simison) [inline client/my-sites/pages/page/index.js:399]: Unrelated, just adding separator here.

File: apps/full-site-editing/full-site-editing-plugin/coming-soon/coming-soon.php [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	if ( ! isset( $post ) ) {
+		return false;
+	}
+
+	// Allow anonymous previews
+	if ( isset( $_GET['preview'] ) ) {
+		return false;
+	}
+
+	return ( (int) get_option( 'page_on_front' ) === $post->ID );
```
Comment (simison) [inline apps/full-site-editing/full-site-editing-plugin/coming-soon/coming-soon.php:27]: This gating can be removed, we can show the coming soon on all pages and not just on frontpage.

**Comment by simison** [comment]
This has served its purpose, closing. :-)

**Pull request closed without merging**